### ISSUE TITLE
Error when invoking Lambda after adding the AWSSDK.SQS library

### ISSUE THREAD

**Issue by vbond007** [aws/aws-sdk-net#1087]
Hi,

Testing a .Net Core 2.1 Lambda triggered by a S3 event, everything was fine until I add the AWSSDK.SQS library (the purpose was to create a lambda reading a file and filling a sqs queue).

If I just create the sample aws lambda in visual studio based on S3 event and deploy it. It works fine.
If then I just add the AWSSDK.SQS library from nuget, without doing any modification in the code, and deploying it, then it fails with following error:

{
  "errorType": "LambdaException",
  "errorMessage": "An exception was thrown when the constructor for type 'AWSLambdaFileUpload.Function' was invoked. Check inner exception for more details.",
  "cause":   {
    "errorType": "TargetInvocationException",
    "errorMessage": "Exception has been thrown by the target of an invocation.",
    "stackTrace": [
      "at System.RuntimeTypeHandle.CreateInstance(RuntimeType type, Boolean publicOnly, Boolean wrapExceptions, Boolean& canBeCached, RuntimeMethodHandleInternal& ctor)",
      "at System.RuntimeType.CreateInstanceSlow(Boolean publicOnly, Boolean wrapExceptions, Boolean skipCheckThis, Boolean fillCache)"
    ],
    "cause":     {
      "errorType": "TypeLoadException",
      "errorMessage": "Method 'get_ServiceMetadata' in type 'Amazon.S3.AmazonS3Client' from assembly 'AWSSDK.S3, Version=3.3.0.0, Culture=neutral, PublicKeyToken=885c28607f98e604' does not have an implementation."
    }
  }
}

So to summarize:
- .Net core 2.1
- Visual Studio 2017 up to date
- Create a simple aws lambda project and choose the "Simple S3 function" blueprint
- Deploy it and invoke it--> it works fine
- Add the AWSSSD.SQS library from Nuget (version 3.3.3.21)
- Deploy it (it works), Invoke it --> the error above

Am I doing something wrong?
Is it a known issue?

Thank you for your help
